Network-Office/src/main/java/dev/office/networkoffice/chat/controller/ChatMessageController.java

```diff
@@ -0,0 +1,25 @@
+package dev.office.networkoffice.chat.controller;
+
+import dev.office.networkoffice.chat.controller.docs.ChatMessageApiDocs;
+import dev.office.networkoffice.chat.dto.response.ChatMessageResponse;
+import dev.office.networkoffice.chat.service.ChatMessageService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1/chat-rooms")
+@RequiredArgsConstructor
+public class ChatMessageController implements ChatMessageApiDocs {
+
+    private final ChatMessageService chatMessageService;
+
+    @GetMapping("/{roomId}/messages")
+    public List<ChatMessageResponse> getMessageList(@PathVariable Long roomId) {
+        return chatMessageService.loadChatMessageList(roomId);
+    }
+}
```

Comment:
```suggestion
@RestController
@RequestMapping("api/v1/chat-rooms")
@RequiredArgsConstructor
public class ChatMessageController implements ChatMessageApiDocs {

    private final ChatMessageService chatMessageService;

    @GetMapping("{roomId}/messages")
    public List<ChatMessageResponse> getMessageList(@PathVariable Long roomId) {
        return chatMessageService.loadChatMessageList(roomId);
    }
}
```

다른 API처럼 맨 앞 `/`는 없는 상태로 작성하는 것이 좋을 것 같아요!

Reply:
아하 알겠습니당 😊